extensions/wikia/InsightsV2/templates/Insights_subpageList.php

```diff
@@ -94,14 +94,16 @@
 				<?php if ( $pagination ) : ?>
 					<?= $pagination ?>
 				<?php endif ?>
-			<?php elseif (!empty( $flagTypes ) ) : ?>
-				<p>
-					<?= wfMessage( 'insights-list-no-flag-types' )->escaped(); ?>
-				</p>
 			<?php else: ?>
-				<p>
-					<?= wfMessage( 'insights-list-no-items' )->escaped(); ?>
-				</p>
+				<?php if ( $usage == InsightsModel::INSIGHTS_USAGE_ACTIONABLE ): ?>
```

Comment:
I think a cleaner solution would be to perform this logic in the controller and pass a prepared message to the template. Also - use `===` if possible. :)